/* Copyright (C) 2025 Interactive Brokers LLC. All rights reserved. This code is subject to the terms
 * and conditions of the IB API Non-Commercial License or the IB API Commercial License, as applicable. */

syntax = "proto3";

package protobuf;

import "ExecutionFilter.proto";

option java_outer_classname = "ExecutionRequestProto";
option java_package = "com.ib.client.protobuf";
option csharp_namespace = "IBApi.protobuf";

message ExecutionRequest {
    optional int32 reqId = 1;
    optional ExecutionFilter executionFilter = 2;
}
